<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogELSModeSelection</class>
 <widget class="QDialog" name="DialogELSModeSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radiobtn_automatic">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>138</y>
     <width>231</width>
     <height>57</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    车载联机</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radiobtn_standalone">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>248</y>
     <width>227</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    车载独立</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radiobtn_deadhead">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    手工控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblelsmodename">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>46</y>
     <width>193</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车载运行模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>240</y>
     <width>119</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>119</width>
     <height>59</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
